integration-tests/api/__tests__/admin/draft-order.js

```diff
@@ -71,6 +71,88 @@ describe("/admin/draft-orders", () => {
       expect(response.status).toEqual(200)
     })
 
+    it("creates a draft order with a billing address that is an AddressPayload and a shipping address that is an ID", async () => {
+      const api = useApi()
+
+      const payload = {
+        email: "oli@test.dk",
+        billing_address: {
+          first_name: "kap",
+          last_name: "test",
+          country_code: "us",
+        },
+        shipping_address: "oli-shipping",
+        items: [
+          {
+            variant_id: "test-variant",
+            quantity: 2,
+            metadata: {},
+          },
+        ],
+        region_id: "test-region",
+        customer_id: "oli-test",
+        shipping_methods: [
+          {
+            option_id: "test-option",
+          },
+        ],
+      }
+
+      const response = await api
+        .post("/admin/draft-orders", payload, {
+          headers: {
+            Authorization: "Bearer test_token",
+          },
+        })
+        .catch((err) => {
+          console.log(err)
+        })
+
+      expect(response.status).toEqual(200)
+      expect(response.data.draft_order.cart.billing_address_id).toBeDefined()
```

Comment:
See other comment. We could assert on the actual billing_address object as well as the shipping address object. Same goes for the test below.